lib/logflare/backends/adaptor/postgres_adaptor.ex

```diff
@@ -23,8 +15,8 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor do
   alias Logflare.Backends.SourceBackend
   alias Logflare.Backends.SourceDispatcher
   alias Logflare.Buffers.MemoryBuffer
-  alias Logflare.Backends.Adaptor.PostgresAdaptor.Pipeline
-  alias Logflare.Backends.Adaptor.PostgresAdaptor.Repo
+  alias __MODULE__.Pipeline
+  alias __MODULE__.PgRepo
```

Comment:
This type of change can also make searching trickier (search all usages for a given module and would return all places where it's aliased) so non blocking but can reduce capacity to search for code usage

Reply:
sure will adjust